packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -8,7 +8,7 @@ import 'package:ubuntu_wizard/utils.dart';
 import 'app.dart';
 import 'installing_state.dart';
 
-void main(List<String> args) {
+void main(List<String> args) async {
```

Comment:
```suggestion
Future<void> main(List<String> args) async {
```

Reply:
I went to the opposite direction due the [effective Dart recommendation for async top-level main functions, where the Future annotation can (and generally should) be dropped in favor of void[(https://dart-lang.github.io/linter/lints/avoid_void_async.html).

Is there something in the runtime expecting to receive this Future<void> returned by main?

Reply:
This can be a little bit confusing:  (Here we are recommended to make even top level main Future<void>)[https://dart.dev/codelabs/async-await#working-with-futures-async-and-await:~:text=If%20the%20function,%7B%20%C2%B7%C2%B7%C2%B7%20%7D]

Would you have a decisive argument for that?

Reply:
A hidden agenda. :) Later on, we could return a future that lets integration tests know when the app is fully up and running and that way get rid of the ugly static startup callback we're relying on today.

Reply:
Good point. I'll fix that.